src/roktManager.ts

```diff
@@ -88,7 +88,19 @@ export default class RoktManager {
         }
 
         try {
-            return this.kit.selectPlacements(options);
+            let enhancedOptions = options;
+
+            if (this.config) {
```

Comment:
I just realized that changes @alexs-mparticle  will making will break this because we will be passing the Rokt config directly, and not the mp config anymore.  I flagged this with Alex, and he had some ideas about how to get this still, including passing `inDevelopmentMode` as an option as a 3rd parameter.